### ISSUE TITLE
[kws_streaming] The quantization results in significant precision loss for internal state model

### ISSUE THREAD

**Issue by ctwillson** [google-research/google-research#2278]
I use the following code to convert the internal state model into TFLite
```
converter.optimizations = [tf.lite.Optimize.DEFAULT]
converter.representative_dataset = representative_dataset_gen
converter.experimental_new_quantizer = True
converter.experimental_enable_resource_variables = True
converter._experimental_variable_quantization = True#converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS_INT8]
converter.inference_input_type = tf.int8
converter.inference_output_type = tf.int8
```
However, I noticed an accuracy loss of about 10% compare to not converted model. Then, I used a non-stream model for Int8 quantization, and there was no significant accuracy loss. Any idea for this issue? @rybakov 
BTW:Thank you to the author for your work, it has given me a lot of inspiration

**Comment by ctwillson** [comment]
My train script:
```
$CMD_TRAIN --data_url '' --data_dir $DATA_PATH3/ --train_dir $MODELS_PATH/ds_tc_resnet/ --split_data 0 --return_softmax 1 --wanted_words 'silence,noise,tiancaitu' --unknown_percentage 2000 --clip_duration_ms 2000 --preprocess micro --mel_upper_edge_hertz 7600 --how_many_training_steps 10000,10000,5000,5000 --learning_rate 0.001,0.0005,0.0002,0.0001 --window_size_ms 30.0 --window_stride_ms 10.0 --mel_num_bins 40 --dct_num_features 40 --resample 0.15 --alsologtostderr --train 1 --use_spec_augment 1 --time_masks_number 2 --time_mask_max_size 25 --frequency_masks_number 2 --frequency_mask_max_size 7 --pick_deterministically 1 **ds_tc_resnet --activation 'relu' --dropout 0.0 --ds_filters '64, 64, 64' --ds_repeat '1, 1, 1' --ds_residual '0, 1, 1' --ds_kernel_size '11, 13, 15' --ds_stride '1, 1, 1' --ds_dilation '1, 1, 1' --ds_padding "'causal','causal','causal'" --ds_repeat '1, 1, 1' --ds_filter_separable '1, 1, 1' --ds_pool '1, 1, 1'**
```
